EpsilonLights/Inc/Lights.h

```diff
@@ -77,3 +91,13 @@ void reportLightsToCanTask(void const* arg);
 void sendHeartbeatTask(void const* arg);
 
 void updateStrobeLight(void const* arg);
+
+void updateLights1(lightsInfo* x, uint32_t* y);
```

Comment:
Is there a reason for the number 1?

Reply:
if i name it "updateLights", I get this warning about osThreadId strobeLightHandle not being used. Not sure why that happens, or what that even has to do with changing that function name.